src/wm_tts_service_EN.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Or maybe its not time to merge.. 
But ok if a clean up is done later because there is allready 4 file xD